Extensions/xkit_patches.js

```diff
@@ -742,7 +742,67 @@ XKit.extensions.xkit_patches = new Object({
 						avatar: post.blog.avatar[post.blog.avatar.length - 1].url,
 						tags: post.tags.join(","),
 					};
+				},
+			};
+
+			XKit.interface.async_form_key = async function() {
+				const request = await fetch('https://www.tumblr.com/settings/dashboard');
+				const meta_tag = (await request.text()).match(
```

Comment:
In my opinion it would be nice to have an example of the expected input format for this RegExp in a comment but I also understand the school of thought where this can be self-documenting and have no need for an additional comment that might get out of step with this code. No pressure to add a comment, this is a well-written, straightforward regex.

Reply:
comments are bad in this case, because only the most exhaustive comment wouldn't fail to capture the nuances of the regex. one thing I do want to write down for posterity is that i started ignoring unquoted attributes intentionally because they make the regex super complicated and tumblr's never going to use them anyway.